### DISCUSSION TITLE
mac OBS virtual camera latest compatibility data ?

### DISCUSSION THREAD

**Discussion by sq3-Amandeep** [obsproject/obs-studio#5517] [Development Questions] [answered]
Hi there,
I want to query regarding the latest OBS mac Virtual camera compatibility among different applications with intel/M1 based machines as apple officially didn’t provide any specific entitlement for DAL plug-in due to which multiple applications don’t support Virtual-Camera plug-in. I had gone through [compatibility](https://github.com/johnboiles/obs-mac-virtualcam/wiki/Compatibility) on johnboiles/obs-mac-virtualcam wiki but, that is a year ago is there any updated data regarding it. Please provide the latest Compatibility data.

**Comment by sebastian-s-beckmann** [comment, marked as answer]
The newest data is this guide: https://github.com/obsproject/obs-studio/wiki/MacOS-Virtual-Camera-Compatibility-Guide

**Reply by sq3-Amandeep** [reply]
Thanks @gxalpha for answer

**Comment by mrwonko** [comment]
The link is dead, where can I find the guide now?

I found a troubleshooting guide at <https://obsproject.com/kb/virtual-camera-troubleshooting>, is that the replacement?

**Reply by sebastian-s-beckmann** [reply]
Yes, it is.

**Comment by rewulff** [comment]
Another issue is the by default activated portrait mode on virtual cam without any chance to disable it.

There sems to be a workaround from apple

https://support.apple.com/en-us/108387

Has anyone checked this already?

**Reply by sebastian-s-beckmann** [reply]
That link is not relevant to your issue. OBS already uses a modern virtual camera.

Either way this appears to be a support request, for which we don't use GitHub Discussions.
Please make sure to visit the support sections on our [Discord server](https://discord.gg/obsproject) or [Forums](https://obsproject.com/forum).

Thank you!

**Reply by rewulff** [reply]
Sorry, you are right. 

The topic is allready in the Forum:
https://obsproject.com/forum/threads/virtual-camera-window-capture-out-of-focus-blurry.173139/
